backend/src/router/hazardtypes.ts

```diff
@@ -1,12 +1,15 @@
 import express from 'express';
 import controller from '../controllers/hazardtypes';
 
+import extractJWT from 'middlewares/extractJWT';
+
 const router = express.Router();
 
-router.post('/create', controller.createHazardType);
-router.get('/all', controller.getAllHazardTypes);
-router.get('/:id', controller.getHazardTypeById);
-router.patch('/update/:id', controller.updateHazardType);
-router.delete('/delete/:id', controller.deleteHazardType);
+router.post('/create',extractJWT, controller.createHazardType);
+router.get('/all',extractJWT, controller.getAllHazardTypes);
+router.get('/:id',extractJWT, controller.getHazardTypeById);
+router.patch('/update/:id',extractJWT, controller.updateHazardType);
+router.delete('/delete/:id',extractJWT, controller.deleteHazardType);
+
```

Comment:
I don't understand how extractJWT middle water is going to allow only admin to before CRUD operations on the hazardTypes.

Also the get routes for the hazardTypes should be accessible by anyone.